plugins/imageverifier/plugin.go

```diff
@@ -0,0 +1,45 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package imageverifier
+
+import (
+	"time"
+
+	"github.com/containerd/containerd/pkg/imageverifier/bindir"
+	"github.com/containerd/containerd/plugin"
+)
+
+// Register default image verifier service plugin
+func init() {
+	plugin.Register(&plugin.Registration{
+		Type:   plugin.ImageVerifierPlugin,
+		ID:     "bindir",
+		Config: defaultConfig(),
+		InitFn: func(ic *plugin.InitContext) (interface{}, error) {
+			cfg := ic.Config.(*bindir.Config)
+			return bindir.NewImageVerifier(cfg), nil
+		},
+	})
+}
+
+func defaultConfig() *bindir.Config {
+	return &bindir.Config{
+		BinDir:             "/opt/containerd-image-verifier/bin",
```

Comment:
On Linux, maybe `/opt/containerd/image-verifier`?  While I don't think this would work easily with the [managed opt plugin](https://github.com/containerd/containerd/blob/main/docs/managed-opt.md), we do already have `/opt/containerd` and it'd be nice in the future to possibly use this mechanism to install verifiers.

For Windows: we should either specify a relevant Windows directory or drop Windows support for the verifier for now (and address in a follow-up?).

Reply:
I need to do some research on Windows file layouts (I have ~no experience with Windows) but this sounds like just a few lines to add Windows support to this PR.

Reply:
Done in 7cccbad9ec0ce2ec82050aa4a772a0e9cb3a297b